src/v2/orders.ts

```diff
@@ -0,0 +1,130 @@
+import { Client } from '../client'
+import { BaseError } from '../errors'
+import { UriHelper } from '../uri-helper'
+import { ThBaseHandler } from '../base'
+
+
+export interface OrdersOptions {
+    user?: string
+    base?: string
+}
+
+export interface OrdersResponse {
+    data: Order[]
+    metadata: Record<string, unknown>
+    next?: () => Promise<OrdersResponse>
+}
+
+export interface OrdersQuery {
+    limit?: number
+    uri?: string
+    query?: {
+        deleted?: boolean
+        active?: boolean
+        extended?: boolean
+        location?: string
+    }
+}
+
+declare type RecurringFormatType = 'scheduled' | 'unscheduled' | 'oneclick'
+declare type ReturnFormatType = 'refundable' | 'capturable' | 'cancellable'
+
+export interface Order {
```

Comment:
This is the basic info of an order, that would make sense for the listing. But an order is made of a lot of different components, like transaction, payments, etc. Adding here the doc:
https://unz.atlassian.net/wiki/spaces/TA/pages/1139474576/Order+data+model+detailed+draft

Reply:
Ok, stupid of me.. I only thought it was order part that was there.